src/marqo/core/structured_vespa_index/structured_vespa_index.py

```diff
@@ -375,15 +382,18 @@ def to_vespa_query(self, marqo_query: MarqoQuery) -> Dict[str, Any]:
                 if root_modifier_field not in self._marqo_index.score_modifier_fields_names:
                     raise InvalidFieldNameError(
                         f'Index {self._marqo_index.name} has no score modifier field {modifier.field}. '
-                        f'Available score modifier fields are: {", ".join(self._marqo_index.score_modifier_fields_names)}'
+                        f'Available score modifier fields are: '
+                        f'{", ".join(self._marqo_index.score_modifier_fields_names)}'
                     )
 
-        if isinstance(marqo_query, MarqoTensorQuery):
+        # Hybrid must be checked first since it is a subclass of Tensor and Lexical
```

Comment:
In this case, I think it makes sense to make MarqoHybridQuery inherits from MarqoQuery directly.

Reply:
the problem is that it needs its attributes as a subclass of both `MarqoTensorQuery` and `MarqoLexicalQuery` to be slotted into these functions seamlessly